doc/md_src_files/040_stakeholders-scope.md

```diff
@@ -23,12 +23,61 @@ Relatedly, C-level executives and public policy professionals often make, shape,
+In this paper, we model the decision of “With what priority should the organization take action on a given vulnerability management work unit?” to be agnostic to whether or not a patch is available. 
+A unit of work means either remediating the vulnerability, such as applying a patch or deploying a mitigation. Both remediation and mitigations often address multiple identified vulnerabilities. 
 
-To further clairify terms, "Remediaton occurs when the vulnerability is eliminated or removed. Mitigation occurs when the impact of the vulnerability is decreased without reducing or eliminating the vulnerability." [@dodi_8531_2020, section 3.5] Examples of remediation includes, applying patches, fixes and upgrades; or removing the vulnerable software or system from operation. Mitigating acions may include, software configuration changes, adding firewall ACLs or otherwise limiting the system's exposure to reduce the risk of the impact of the vulnerability; or accepting the risk.
+The unit of work may be different for different stakeholders.
+The units of work can also change as the vulnerability response progresses through a stakeholder's process.
+We elucidate these ideas with the following examples.
 
-### Supplying Patches
+### Vendor Units of Work
+
+On the input side of the vendor process, vendors typically receive reports of vulnerabilities in one or more versions of their product. 
+Part of the vendor's task on initial report intake is to resolve the initial report into a set of products and versions that are affected by the reported vulnerability. 
+On the output side, vendors provide fix bundles for affected products.
+
+Our working assumption is that for SSVC purposes, the vendor's unit of work is the tuple of (vulnerability, affected product). 
+This implies the need for vendors to be able to resolve whatever they receive to that level of granularity in order to make best use of SSVC.
+
+There are a variety of ways a vendor might need to resolve the set of affected products. For example they might
+
+* recognize, on further investigation of the initial report, that additional versions of the product are affected
+* discover that other products are affected due to code sharing or even consistent programmer error across products
+* receive reports of vulnerabilities in third party libraries they include in one or more of their products
+* receive fix bundles for third party libraries used in one or more of their products (where a fix bundle might resolve multiple vulnerabilities or add new features)
+
+We note that SBOM seems well-placed to aid in that resolution process for the third party library scenarios.
```

Comment:
Add citation to SBOM. [@manion2019sbom] is already in the bib file. 

Also, it seems worth comparing these considerations to the CNA rules for assigning CVE-IDs. I think they are largely compatible with the CNA rules? Will need to be cited and added to bib. 
```
@techreport{mitre2020cna,
  title={{CVE} Numbering Authority {(CNA)} rules},
  author={{CVE Board}},
  year={2020},
  month = feb,
  number = {ver. 3.0},
  address = {Bedford, MA},
  institution={MITRE},
  url = {https://cve.mitre.org/cve/cna/rules.html}
}
```

Reply:
I'm not an expert on CNA & CVE rules.  Please assign this to Madison or Art.

Sorry!

Reply:
fixed as of e3226fba3c1ffa157467cb15143e4b4765c0163a